shallow_backup/reinstall.py

```diff
@@ -74,9 +74,11 @@ def reinstall_configs_sb(configs_path):
 			copyfile(path_to_backup, dest_path)
 
 	# reinstall crontab
-	with open("{0}/crontab.txt".format(configs_path), "r") as f:
+	with open(os.path.join(configs_path, "/crontab.txt"), "r") as f:
 		for x in f:
-			cmd = "(crontab -l ; echo \"{0}\") | sort - | uniq - | crontab -".format(x)
+			# the replace sanitizes the crontab line of any present " characters.
+			cmd = "(crontab -l ; echo \"{0}\") | sort - | uniq - | crontab -".format(
```

Comment:
Use f-strings instead of format calls.

Reply:
Although I don't fully understand the purpose of this subsequence.

Reply:
The subsequence makes sure that that only non existing lines are added to the crontab and existing lines are kept. The escaping of `"` characters is done because it would break the bash code.